core/db/migrate/20160225152313_remove_tax_rate_from_shipping_rate.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveTaxRateFromShippingRate < ActiveRecord::Migration
+  def up
+    Spree::ShippingRate.find_each do |shipping_rate|
+      tax_rate_id = shipping_rate.tax_rate_id
+      if tax_rate_id
+        tax_rate = Spree::TaxRate.unscoped.find_by(shipping_rate.tax_rate_id)
+        shipping_rate.taxes.create(
+          tax_rate: tax_rate,
+          amount: tax_rate.compute_amount(shipping_rate)
+        )
+      end
+    end
```

Comment:
It would be nice to wrap this in a [`say_with_time` block.](http://apidock.com/rails/ActiveRecord/Migration/say_with_time) That way we still get some feedback when running the migration on how long this section took, what it's doing, and how many rows were affected.